packages/components/Menu/SPEC.md

```diff
@@ -293,9 +293,9 @@ export interface MenuListTokens extends LayoutTokens, IBackgroundColorTokens {
 
   /**
    * Corner radius of the menu list
-   * @platform android
+   * @platform android macos
    */
-  cornerRadius?: number;
+  borderRadius?: number;
```

Comment:
breaking change?

Reply:
Mostly for Android since we didn't support this elsewhere till now but yes. I should do a sanity check whether it's in use..

Reply:
@rohanprasadofficial / @ayush547 is Android Menu in use right now?

Reply:
I looked internally and didn't see any uses, so I think I'm going to resolve this. We can revert if needed.